src/agentscope/models/__init__.py

```diff
@@ -64,6 +67,7 @@
     "ZhipuAIEmbeddingWrapper",
     "LiteLLMChatWrapper",
     "YiChatWrapper",
+    "StableDiffusionTxt2imgWrapper",
```

Comment:
Since the SD API also provides an img2img interface, I understand that both txt2img and img2img represent image synthesis, I'm concerned that naming the wrapper StableDiffusionImageSynthesisWrapper to represent text-to-image functionality might lead to confusion in the future if we decide to add an image-to-image wrapper.

Reply:
> Please change name to `StableDiffusionImageSynthesisWrapper`

Since the SD API also provides an img2img interface, I think that both txt2img and img2img represent image synthesis, I'm concerned that naming the wrapper StableDiffusionImageSynthesisWrapper to represent text-to-image functionality might lead to confusion in the future if we decide to add an image-to-image wrapper.